models/message.go

```diff
@@ -73,3 +76,30 @@ type AttachmentField struct {
 	Title string `json:"title"`
 	Value string `json:"value"`
 }
+
+type AttachmentActionType string
+
+const {
+	AttachmentActionTypeButton AttachmentActionType = "button"
+}
+
+// AttachmentAction are action buttons on message attachments
+type AttachmentAction struct {
+	Type AttachmentActionType `json:"type"`
```

Comment:
What happened to the golang formatting on this one?

Reply:
I did it in the browser 🙈